lib/DonorsList/AddDonorButton.js

```diff
@@ -0,0 +1,60 @@
+import { map } from 'lodash';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Pluggable } from '@folio/stripes/core';
+
+import {
+  initialFilters,
+  modalLabel,
+  pluginVisibleColumns,
+  resultsPaneTitle,
+  searchableIndexes,
+  visibleFilters,
+} from './constants';
+
+const AddDonorButton = ({ fetchDonors, fields, stripes, name }) => {
+  const addDonors = (donors = []) => {
+    const addedDonorIds = new Set(fields.value);
+    const newDonorsIds = map(donors.filter(({ id }) => !addedDonorIds.has(id)), 'id');
+
+    if (newDonorsIds.length) {
+      fetchDonors([...addedDonorIds, ...newDonorsIds]);
```

Comment:
the `fetchDonors` should be called smth like `onAddDonor`, btn doesn't fetch anything

Reply:
updated